cmd/libs/go2idl/generator/generator.go

```diff
@@ -110,6 +110,10 @@ type Generator interface {
 	// Generators.)
 	Init(*Context, io.Writer) error
 
+	// Finalize should write finish up functions, and any other content that's not
```

Comment:
added more comment there. Basically it should be finish up code, the example is the exact thing I did here, generating one function for all types. Finalize used to close the function block. Added some comment to clarify it.